frontend/src/components/navigation/NavHeader.tsx

```diff
@@ -188,8 +189,9 @@ export default function NavHeader({
                 </div>
               ) : (
                 <Link
-                  to="/projects/secret-management/$projectId/secrets/$envSlug"
+                  to="/organizations/$orgId/projects/secret-management/$projectId/secrets/$envSlug"
                   params={{
+                    orgId: currentOrg?.id || "",
                     projectId: currentProject.id,
                     envSlug: routerEnvSlug || ""
```

Comment:
When logging in, i am getting this error

<img width="2525" height="1406" alt="Image" src="https://github.com/user-attachments/assets/71a9f0d3-f96e-44a2-9c80-1d6cea9a5e1b" />